.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,62 @@
----
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: Feature request
-assignees: ''
 
----
+## Problem
+<!-- Describe what you are trying to achive. You can include proposed solution.  -->
 
-### What I'm trying to achieve
-…
 
-### Describe a proposed solution
-...
+___
 
-### Other solutions I've tried and won't work
-…
+## 1. General Assumptions
+<!--   -->
 
-### Screenshots or mockups
-<!-- Please provide any illustrations that could help others understand the problem or the proposed solution. -->
+### Acceptance Criteria 
+<!-- Conditions that a feature must satisfy to be approved by QA 
+- Functional use case:
+  - Permissions: As USER/APP with permision `XYZ' I should be able to 'action" 
+  - Webhooks: Webhook `name` event should be sent for every updated `instance name`
+  - Support for old versions: After updating, it is possible to retrieve data created in older (3.1, 3.5, 3.7) Saleor versions.
+  - Error handling: Should be handled according to 'policy_name' 
+  - Other use-cases: ID or externalReference can be used to identify object
+- Non-functional use case 
+  - X objects can be updated in one mutation
+-->
+
+- [ ] 
+- [ ] 
+
+ 
+## 2. API changes
+### Mutations
+```
+```
+### Types
+```
+```
+
+## 3. Database changes
+<!-- New models or changes in existing models  -->
+
+
+## 4. UML diagrams
+<!-- You can render UML diagrams using [Mermaid](https://mermaidjs.github.io/). -->
+```mermaid
+```
+
+## 5. To Do list
```

Comment:
We could use github's tasklist here. By having this, we could really easy convert the item from the list into sub-task (separate github issue, linked to the feature)
https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists#creating-tasklists-with-markdown